index.d.ts

```diff
@@ -35,3 +41,5 @@ export declare const koaMiddleware: (
 export declare const koaV1Middleware: (
   options?: IMiddlewareOptions,
 ) => GeneratorFunction
+
+export declare const hapiPlugin: IHapiPlugin
```

Comment:
Looks like `IHapiPlugin` contains internal implementation details specific to hapi. Those details, except for options object, aren't intended to be used by `cls-rtracer` users. Maybe we can reference hapi's interfaces (see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/hapi/v17/index.d.ts#L157)?

@keenondrums could you take a look?

Reply:
Makes sense, let me take a look